llvm/test/CodeGen/AMDGPU/GlobalISel/flat-scratch.ll

```diff
@@ -1510,4 +1510,243 @@ bb:
   ret void
 }
 
+define amdgpu_gs void @sgpr_base_large_offset(ptr addrspace(1) %out, ptr addrspace(5) inreg %sgpr_base) {
```

Comment:
We should probably remove this file, and add extra run lines to the other file instead. But that would be a separate patch.